### PR TITLE
Fix grammar issues

### PR DESCRIPTION
**Pull request by drthanwho**
## Proposed change

Just changing some text to fix any grammatical mistakes. Also added/clarified a _click here_ step tha was missing. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/alexa.intent.markdown [hunk 1]
```diff
@@ -17,9 +17,9 @@ The built-in Alexa integration allows you to integrate Home Assistant into Alexa
 
 ### Requirements
 
-- Amazon Developer Account. You can sign on [here](https://developer.amazon.com).
-- An [AWS account](https://aws.amazon.com/free/) is need if you want to use Alexa Custom Skill API. Part of your Alexa Custom Skill will be hosted on [AWS Lambda](https://aws.amazon.com/lambda/pricing/). However you don't need worry the cost, AWS Lambda allow free to use up to 1 millions requests and 1GB outbound data transfer per month.
-- The Alexa Custom Skill API also needs your Home Assistant instance to be accessible from the internet via HTTPS on port 443 using a certificate signed by [an Amazon approved certificate authority](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport), this is so account linking can take place. Read more on [our blog](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) about how to set up encryption for Home Assistant. When running Home Assistant using the [Duck DNS](/addons/duckdns/) add-on is the easiest method.
+- Amazon Developer Account. You can sign in [here](https://developer.amazon.com).
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]: This change is incorrect. This is the requirements listing, and tell you where to get an account. You sign up or on for a new account, but you sign-in into an existing account.

Please revert this change.

File: source/_integrations/alexa.intent.markdown [hunk 2]
```diff
@@ -17,9 +17,9 @@ The built-in Alexa integration allows you to integrate Home Assistant into Alexa
...
+- An [AWS account](https://aws.amazon.com/free/) is needed if you want to use the Alexa Custom Skill API. Part of your Alexa Custom Skill will be hosted on [AWS Lambda](https://aws.amazon.com/lambda/pricing/). However you don't need to worry about the cost, as AWS Lambda allows for free to use up to 1 million requests and 1GB outbound data transfer per month.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
- An [AWS account](https://aws.amazon.com/free/) is needed if you want to use the Alexa Custom Skill API. Part of your Alexa Custom Skill will be hosted on [AWS Lambda](https://aws.amazon.com/lambda/pricing/). However, you don't need to worry about the cost, as AWS Lambda allows for free to use up to 1 million requests and 1GB outbound data transfer per month.
```

File: source/_integrations/alexa.intent.markdown [hunk 3]
```diff
@@ -17,9 +17,9 @@ The built-in Alexa integration allows you to integrate Home Assistant into Alexa
...
+- The Alexa Custom Skill API also needs your Home Assistant instance to be accessible from the internet via HTTPS on port 443 using a certificate signed by [an Amazon approved certificate authority](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport). This is so account linking can take place. Read more on [our blog](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) about how to set up encryption for Home Assistant. When running Home Assistant OS using the [Duck DNS](/addons/duckdns/) add-on is the easiest method.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
- The Alexa Custom Skill API also needs your Home Assistant instance to be accessible from the internet via HTTPS on port 443 using a certificate signed by [an Amazon approved certificate authority](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport). This is so account linking can take place. Read more on [our blog](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) about how to set up encryption for Home Assistant. When running Home Assistant OS or Supervised, using the [Duck DNS](/addons/duckdns/) add-on is the easiest method.
```

File: source/_integrations/alexa.intent.markdown [hunk 4]
```diff
@@ -68,8 +68,8 @@ Next you need to create a Lambda function.
 - Click `Create function`, select `Author from scratch`, then input a `Function name`.
 - Select *Python 3.6* or *Python 3.7* as `Runtime`.
 - Select *Use an existing role* as `Execution role`, then select the role you just created from the `Existing role` list.
-- Click `Create function`, then you can configuration detail of Lambda function.
-- Under `Configuration` tab, expand `Designer`, then click `Alexa Skills Kit` in the left part of the panel to add a Alexa Skills Kit trigger to your Lambda function.
+- Click `Create function`, then you can configure the details of the Lambda function.
+- Under `Configuration` tab, expand `Designer`, then click on `+ Add trigger` in the left part of the panel and select `Alexa Skills Kit` from the dropdwon list to add an Alexa Skills Kit trigger to your Lambda function.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
- Under the `Configuration` tab, expand `Designer`, then click on `+ Add trigger` in the left part of the panel and select `Alexa Skills Kit` from the dropdown list to add an Alexa Skills Kit trigger to your Lambda function.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/alexa.intent.markdown (hunk 1 above)
Comment (drthanwho) [reply source/_integrations/alexa.intent.markdown]: I was under the wrong impression you could just log in your current Amazon account. My bad.

**Review by drthanwho — COMMENTED** [review]
I probably didn't apply the suggestions the best way possible 😅 But hope they are all in now. Let's see

**Review by frenck — APPROVED** [review]
Thanks, @drthanwho 👍
